File: Algorithms/Searching/Linear Search/linear-search.go
/*
 * Linear Search
 *
 */

package main

import "fmt"

func LinearSearch(arr []int, k int) int {
	for i := 0; i < len(arr); i++ {
		if arr[i] == k {
			return i
		}
	}
	return -1
}

func main() {
	arr := []int{3, 5, 12, 56, 92, 123, 156, 190, 201, 222}
	fmt.Println(LinearSearch(arr, 12))
}

File: Algorithms/Searching/Linear Search/linear-search.js
/*
 * Linear Search
 *
 */

function LinearSearch(arr, k) {
    for(let i =0; i< arr.length;i++){
        if (arr[i] == k) {
            return i
        }
    }
    return -1
}

console.log(LinearSearch([3, 5, 12, 56, 92, 123, 156, 190, 201, 222], 12))
